docs/site/Model.md

```diff
@@ -918,10 +918,35 @@ If the `jsonSchema` field is missing, you will get an error saying
 
 > You must provide the "jsonSchema" field when define a nested array property'
 
-### Validation Rules
+### Custom Validation Rules and Error Messages
 
-You can also specify the validation rules in the field `jsonSchema`. For
-example:
+You can also specify the validation rules in the `jsonSchema` field of the
+property option and configure them with custom error messages.
+
+For this to work, you should first create a `RequestBodyValidationOptions`
+object and bind it to `RestBindings.REQUEST_BODY_PARSER_OPTIONS` as shown in the
+example below.
+
+```ts
+import {RestApplication, RestBindings, RequestBodyValidationOptions} from '@loopback/rest';
+...
+export class MApplication extends BootMixin(
+  ServiceMixin(RepositoryMixin(RestApplication)),
+) {
+  constructor(options: ApplicationConfig = {}) {
+    super(options);
+
+    const validationOptions: RequestBodyValidationOptions = {
+      $data: true,
+      ajvKeywords: true,
+      ajvErrors: true
+    };
+
+    this.bind(RestBindings.REQUEST_BODY_PARSER_OPTIONS).to({validation: validationOptions});
```

Comment:
We initially set up the options in [here](https://github.com/strongloop/loopback-next/blob/a24ea4f7fb9a0333f16282a8586d3ab75a77bd6c/packages/rest/src/validation/ajv-factory.provider.ts#L54-L60), when the error message is a string, there is no need to configure the options...I am also confused why the options are needed as an object.

Reply:
You will notice until https://github.com/hacksparrow/lb4.new/blob/master/src/application.ts#L27 is uncommented, custom errors don't work as expected.

So, maybe it is actually a bug than bad documentation?

Reply:
@hacksparrow Thanks for adding the default options, then I think we can delete the extra step in doc?